Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM rocker/tidyverse:4.2
+MAINTAINER rokita@chop.edu
+WORKDIR /rocker-build/
+
+RUN RSPM="https://packagemanager.rstudio.com/cran/2022-10-07" \
+  && echo "options(repos = c(CRAN='$RSPM'), download.file.method = 'libcurl')" >> /usr/local/lib/R/etc/Rprofile.site
+  
+COPY scripts/install_bioc.r .
+
+COPY scripts/install_github.r .
+
+### Install apt-getable packages to start
+#########################################
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils dialog
+
+# Add curl, bzip2 and some dev libs
+RUN apt-get update -qq && apt-get -y --no-install-recommends install \
+    curl \
+    bzip2 \
+    zlib1g \
+    libbz2-dev \
+    liblzma-dev \
+    libreadline-dev
+
+# libmagick++-dev is needed for coloblindr to install
+RUN apt-get -y --no-install-recommends install \
+    libgdal-dev \
+    libudunits2-dev \
+    libmagick++-dev
+
+# Required for installing pdftools, which is a dependency of gridGraphics
+RUN apt-get -y --no-install-recommends install \
+    libpoppler-cpp-dev
+
+# Install java
+RUN apt-get update && apt-get -y --no-install-recommends install \
+   default-jdk \
+   libxt6
+   
+# cmakeis needed for ggpubr to install
+RUN apt-get -y --no-install-recommends install \
+    cmake
+
+# install R packages from CRAN
+RUN install2.r \
+	BiocManager \
+  ComplexHeatmap \
+  data.table \
+  ggpubr \
+  ggthemes \
+  msigdbr \
+	optparse \
+	pheatmap \
+	RColorBrewer \
+	survival \
+  survMisc \
+  survminer \
+  tidytext
```

Comment:
These are a good start but I think you also may need others, as you had indicated in your PR: #1 
```
library(flextable)
```

Also, if you think of anything else you might need off the bat which you tend to use or we may need, ie - vroom for loading the large MAF files, feel free to add here and rebuild.